packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -114,7 +114,8 @@ abstract class FlameChartState<T extends FlameChart, V> extends State<T>
   }
 
   void expandRows(int newRows) {
-    for (int i = rows.length; i < rows.length + newRows; i++) {
+    final currentLength = rows.length;
```

Comment:
this could just be 
` for (int i = 0; i < newRows; i++) {`

That said, I think it would be more robust to instead of having an expandRows helper that takes a number of new rows, have a helper that takes then new length of the list as that would prevent any possible bugs added extra un-needed rows because there were already enough rows to fit the content.

Reply:
done